meetings/2020-01-14.md

```diff
@@ -0,0 +1,101 @@
+# OpenJS Foundation Cross Project Council Meeting 2020-01-14
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=kwHp26-16bc
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/444
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Michael Dawson (@mhdawson)
+* Myles Borins (@MylesBorins)
+* Rachel Romoff (@rromoff)
+* Anton Molleda (@molant)
+* Jory Burson (@jorydotcom)
+* Timmy Willison (@timmywil)
+* Jordan Harband (@LJHarb)
+* Christian Bromann (@christian-bromann)
+* Sendil Kumar (@sendilkumarn)
+* Ben Michel (@obensource)
+* Tierney Cyren (@bnb)
+* Mike Samuel (@mvsamuel)
+
```

Comment:
I joined but dropped 30 mins into the meeting :)

Reply:
You were there though! No change needed here.